Core/Rewriters/MoqUsingDirectiveRewriter.cs

```diff
@@ -42,7 +42,21 @@ public class MoqUsingDirectiveRewriter : RewriterBase
         currentUsings = currentUsings.Add (Formatter.MarkWithFormatAnnotation (mockRepositoryAlias));
       }
 
-      return node.WithUsings (currentUsings);
+      var systemUsings = currentUsings.Where (u => IsSystemUsingDirective (u) && !HasAlias (u)).OrderBy (u => u.ToString());
+      var otherUsings = currentUsings.Where (u => !IsSystemUsingDirective (u) && !HasAlias (u)).OrderBy (u => u.ToString());
+      var aliasUsings = currentUsings.Where (HasAlias).OrderBy (u => u.ToString());
```

Comment:
what about `using static` directives?

Reply:
just updated the code, the new order is:
1. system usings
2. other usings
3. static usings
4. alias